tableauserverclient/models/connection_item.py

```diff
@@ -52,6 +69,7 @@ def from_response(cls, resp, ns) -> List["ConnectionItem"]:
             connection_item.server_address = connection_xml.get("serverAddress", None)
             connection_item.server_port = connection_xml.get("serverPort", None)
             connection_item.username = connection_xml.get("userName", None)
+            connection_item._query_tagging = connection_xml.get("queryTaggingEnabled", None)
```

Comment:
Just double checking we don't need to use `string_to_bool()` here like we did on `connection_item.embed_password`

Reply:
good catch. Added that and tests to cover it.